packages/palette/src/elements/Box/Box.ios.tsx

```diff
@@ -0,0 +1,77 @@
+// @ts-ignore
+import React from "react"
+import { styled as primitives } from "../../platform/primitives.ios"
```

Comment:
The only reason I needed to duplicate this file was to hardcode `primitives.ios` here. Otherwise the web version of Box would break rendering using `react-native-web`

Reply:
This is fine for now but we should figure this out as to avoid duplication